src/main/java/uk/gov/companieshouse/certifiedcopies/orders/api/interceptor/UserAuthenticationInterceptor.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.companieshouse.certifiedcopies.orders.api.interceptor;
+
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+import uk.gov.companieshouse.certifiedcopies.orders.api.util.EricHeaderHelper;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.springframework.http.HttpStatus.UNAUTHORIZED;
+import static uk.gov.companieshouse.certifiedcopies.orders.api.logging.LoggingUtils.*;
+
+public class UserAuthenticationInterceptor extends HandlerInterceptorAdapter {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(APPLICATION_NAMESPACE);
```

Comment:
The Logger should be obtained from the LoggingUtils class LoggingUtils.getLogger() - see CertifiedCopiesItemController for the usage examples